app/assets/stylesheets/mobile/mobile.scss

```diff
@@ -176,7 +176,7 @@ footer {
     border-radius: 3px 3px 0 0;
 
     border: {
-      bottom: 1px solid #ccc;
+      bottom: 1px solid $border-medium-grey;
```

Comment:
`$border-medium-grey` existed before, I'm only using it instead of the hardcoded color.

But I agree with you about the naming. I tried to use better names for new variables I created, but I didn't rename existing variables. I didn't want to change too much here, the PR is already big enough. I only used the existing variables instead of hardcoded colors, so I need less css-hacks here. I think the renaming should be done in another PR?